core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -222,13 +222,20 @@ pub async fn verify_tx(tx_hash: &str, network: Network) -> Result<PaymentDetails
 }
 
 fn get_provider(network: Network) -> Provider {
-    let provider: Provider = match env::var("ZKSYNC_RPC_ADDRESS").ok() {
+    let provider: Provider = match get_rpc_from_env(network) {
         Some(rpc_addr) => Provider::from_addr(rpc_addr),
         None => Provider::new(get_zk_network(network)),
     };
     provider.clone()
 }
 
+fn get_rpc_from_env(network: Network) -> Option<String> {
```

Comment:
Little bit more descriptive name
```suggestion
fn get_rpc_address_from_env(network: Network) -> Option<String> {
```

Reply:
good one! updated it to `addr` to not make the name too long ( and be consistent with zksync's `get_rpc_addr` )